data/newMigrations/005-program_providers.js

```diff
@@ -0,0 +1,26 @@
+exports.up = function (knex) {
+    return knex.schema.createTable('program_providers', (tbl) => {
+      tbl.increments('id').primary();
+      tbl
+      .integer('program_id', 128)
+      .unsigned()
+      .notNullable()
+      .references('program_id')
+      .inTable('programs')
+      .onUpdate('CASCADE')
```

Comment:
I don't think we should be using CASCADE for onDelete or onUpdate. We should probably switch this to RESTRICT